cli/flox-rust-sdk/src/environment.rs

```diff
@@ -17,7 +17,10 @@ pub static NIX_BIN: &str = env!("NIX_BIN");
 /// For flox specifically, set Nix-provided defaults for certain
 /// environment variables that we know to be required on the various
 /// operating systems.
-pub fn default_nix_subprocess_env() -> HashMap<String, String> {
+///
+/// * `set_all` - Set all environment variables irrespective of
+///               their presence in the current environment.
+pub fn default_nix_subprocess_env(set_all: bool) -> HashMap<String, String> {
     let mut env_map: HashMap<String, String> = HashMap::new();
 
```

Comment:
suggestion: remove `set_all` parameter
the only time we call this function we call it with `true` (i think we recently removed the only usage of theis being `false`)

Reply:
In fact this function is currently only used by `activate`.
Is the idea to add these commands to all subprocesses or only provide them in environment shells?